SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -266,15 +273,29 @@ void FLocalDeploymentManager::TryStartLocalDeployment(FString LaunchConfig, FStr
 	FSpatialGDKServicesModule& GDKServices = FModuleManager::GetModuleChecked<FSpatialGDKServicesModule>("SpatialGDKServices");
 	TWeakPtr<SSpatialOutputLog> SpatialOutputLog = GDKServices.GetSpatialOutputLog();
 
-	RuntimeProcess->OnOutput().BindLambda([&, SpatialOutputLog](const FString& Output) {
-		SpatialOutputLog.Pin()->FormatAndPrintRawLogLine(Output);
+	RuntimeProcess->OnOutput().BindLambda(
+		[&RuntimeLogFileHandle = RuntimeLogFileHandle, &bStartingDeployment = bStartingDeployment, SpatialOutputLog](FString& Output) {
```

Comment:
`&RuntimeLogFileHandle = RuntimeLogFileHandle, &bStartingDeployment = bStartingDeployment` could just be `&RuntimeLogFileHandle, &bStartingDeployment` ?

Reply:
Nah they are class variables so can't be scoped like that afaik. Doesn't compile